airflow/providers/google/cloud/links/dataproc.py

```diff
@@ -273,11 +273,12 @@ def persist(
         batch_id: str,
         project_id: str,
         region: str,
+        tags: list[str] | None = None,
     ):
         operator.xcom_push(
             context,
             key=DataprocBatchLink.key,
-            value={"batch_id": batch_id, "region": region, "project_id": project_id},
+            value={"batch_id": batch_id, "region": region, "project_id": project_id, "tags": tags},
```

Comment:
Does the batch URL contain network tags?

Reply:
@moiseenkov I am verifying this, will confirm in a couple of days.